libraries/redcore/api/hal/hal.php

```diff
@@ -671,6 +679,11 @@ public function apiCreate()
 			$this->setData('id', $model->getState($model->getName() . '.id'));
 		}
 
+		if (method_exists($model, 'getErrors') && !empty($model->getErrors()))
```

Comment:
Fatal error: Can't use method return value in write context in /home/travis/build/redCOMPONENT-COM/redCORE/tests/joomla-cms3/libraries/redcore/api/hal/hal.php on line 682